site/docs/spec.md

```diff
@@ -254,6 +255,33 @@ Notes:
 
 1. Technically, data files can be deleted when the last snapshot that contains the file as “live” data is garbage collected. But this is harder to detect and requires finding the diff of multiple snapshots. It is easier to track what files are deleted in a snapshot and delete them when that snapshot expires.
 
+#### Deletion Files
+
+Deletion files are files that indicate deletions of pre-existing rows to be applied to the dataset at read time. Deletion files may either specify rows by column value or by file name and row position.
+
+1. The file and position based deletion file has the schema as following:
+```
+{
+  filename string,
+  position long
+}
```

Comment:
Could you update this to be like the other schemas defined in the spec? It should be a table with a description of each field, Iceberg types, and should include field IDs.

Reply:
Also, these fields are `required`.

Reply:
Sure, no problem.

Reply:
You mean json format, right?

Reply:
No, I meant table. Look at the other struct type definitions and copy those. The table format makes it easy to read.

Reply:
Just updated. Plus I use the field ID 600 and 601.